packages/alea-frontend/pages/api/access-control/get-all-resourceacces-pairs.ts

```diff
@@ -0,0 +1,14 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { checkIfGetOrSetError, executeAndEndSet500OnError, getUserIdOrSetError } from "../comment-utils";
+import { isMemberOfAcl } from "../acl-utils/acl-common-utils";
+
+export default async function handler(req : NextApiRequest, res : NextApiResponse){
+    if(!checkIfGetOrSetError(req, res)) return;
+    const userId = await getUserIdOrSetError(req, res);
+    if(!await isMemberOfAcl('sys-admin', userId)){
+        return res.send('not valid');
+    }
+    const query = `SELECT * FROM resourceaccess`;
```

Comment:
Never `SELECT *` See style_guide: 2.6.1

Reply:
shall I retrieve all the fields with their field names?

Reply:
Yes. Specify them even if all fields are needed.